From 7c815e105debcf4fbad2a70753f8b70b7ad118ba Mon Sep 17 00:00:00 2001
From: manthan chauhan <manthanchauhan913@gmail.com>
Date: Wed, 6 May 2020 14:22:46 +0530
Subject: [PATCH] Task 1 (with bonus task)

---
 .gitignore | 1 +
 1 file changed, 1 insertion(+)
 create mode 100644 .gitignore

diff --git a/.gitignore b/.gitignore
new file mode 100644
index 0000000..9f11b75
--- /dev/null
+++ b/.gitignore
@@ -0,0 +1 @@
+.idea/
-- 
2.25.1

