From 5fe43fee83225dbec7336fa4094366891fbf910a Mon Sep 17 00:00:00 2001
From: manthan chauhan <manthanchauhan913@gmail.com>
Date: Wed, 6 May 2020 14:32:52 +0530
Subject: [PATCH] Task 1 (with bonus task)

---
 .gitignore | 1 +
 1 file changed, 1 insertion(+)

diff --git a/.gitignore b/.gitignore
index 9f11b75..331561b 100644
--- a/.gitignore
+++ b/.gitignore
@@ -1 +1,2 @@
 .idea/
+**/*.patch
-- 
2.25.1

